geostore/populate_catalog/task.py

```diff
@@ -63,7 +63,7 @@ def get_collection_href(self, col: Collection, parent_dir: str, is_root: bool) -
         assert not is_root
         return str(col.get_self_href())
 
-    def get_item_href(self, item: Item, parent_dir: str) -> str:
+    def get_item_href(self, item: Item, parent_dir: str) -> str:  # pragma: no cover
```

Comment:
`GeostoreSTACStrategy` `get_item_href` is not called after `normalized_href` is removed; however, this looks like it may be relevant again in the future as the codebase grows (and keeping this for completeness). Excluding this function from coverage, for now.

Reply:
I would not leave dead code in place for "completeness". We'll have the repo history, so it can be recovered if we ever need it. Most likely we never will.

Reply:
It looks like `GeostoreSTACLayoutStrategy` overrides `HrefLayoutStrategy`, and can only be instantiated if it overrides all the abstract methods in the parent class??

Reply:
In that case, how about replacing the method body with `raise NotImplementedError`? That way we'll at least know to remove the coverage pragma once it's used again.